README.md

```diff
@@ -158,6 +158,89 @@ Set `googleAnalytics` in `config.toml` to activate Hugo's [internal Google Analy
 
 Set `disqusshortname` in `config.toml` to activate Hugo's [internal Disqus template](https://gohugo.io/templates/internal/#disqus).
 
+## Staticman
+
+Procedures to start using Staticman:
+
+1. Add Staticman bot/app, depending on your Git service provider.
+    - GitHub: choose either one of the following method
+        + GitHub App: refer to issue
+        [https://github.com/eduardoboucas/staticman/issues/243](https://github.com/eduardoboucas/staticman/issues/243)
```

Comment:
@staticmanapp and `https://api.staticman.net/v2` no logner work, as reported in eduardoboucas/staticman#306.